motionplan/rrt.go

```diff
@@ -39,22 +39,11 @@ type rrtParallelPlannerShared struct {
 
 type rrtMap map[node]node
 
+// TODO would like to rename this to solution since now plans are more of a defined thing.
 type rrtPlanReturn struct {
-	steps   []node
-	planerr error
-	maps    *rrtMaps
-}
-
-func nodesToInputs(nodes []node) [][]referenceframe.Input {
```

Comment:
@nicksanford I think a few of your comments are confusion around the removal of this function.  Its not doing a whole lot, so the places you ask if I'm fixing a bug is just me getting rid of this because its not necessary anymore